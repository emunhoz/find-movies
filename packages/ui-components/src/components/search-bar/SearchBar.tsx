import React from 'react'
import { SearchIcon } from '@commons/images'
import * as S from './SearchBar.style'

interface ISearchBar {
  placeholder?: string
}

export const SearchBar = ({ placeholder }: ISearchBar) => (
  <S.SearchBar>
    <S.SearchIcon><SearchIcon/></S.SearchIcon>
    <S.Input type="text" placeholder={placeholder} />
  </S.SearchBar>
)
