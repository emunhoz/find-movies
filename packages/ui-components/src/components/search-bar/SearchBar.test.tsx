import React from 'react'
import { render } from '../../utils/test-utils'
import { SearchBar } from './SearchBar'

describe('[Component]: SearchBar', () => {
  test('render the component', () => {
    const { asFragment } = render(<SearchBar />)
    expect(asFragment()).toMatchSnapshot()
  })
})