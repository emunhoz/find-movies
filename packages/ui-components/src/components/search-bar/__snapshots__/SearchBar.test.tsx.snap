// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: SearchBar render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ VfWiw"
  >
    <div
      class="sc-gsTCUz kOXcoc"
    >
      <test-file-stub />
    </div>
    <input
      class="sc-dlfnbm jCozXD"
      type="text"
    />
  </div>
</DocumentFragment>
`;
